cypress/e2e/email_scheduled.cy.js

```diff
@@ -0,0 +1,106 @@
+describe("Email Test", () => {
+  it("initiate email campaign", async () => {
+    // have these vars here so it won't affect retry results
+    // (e.g. emails received from previous try being counted in the latest one
+    // as they share the same subject line)
+    const MODE = "email";
+    const CSV_FILENAME = "testfile_email.csv";
+    const NUM_RECIPIENTS = "1";
+    const CUR_DATE = new Date();
+    const DATETIME =
+      CUR_DATE.getDate() +
+      "/" +
+      (CUR_DATE.getMonth() + 1) +
+      "/" +
+      CUR_DATE.getFullYear() +
+      "@" +
+      CUR_DATE.getHours() +
+      ":" +
+      CUR_DATE.getMinutes() +
+      ":" +
+      CUR_DATE.getSeconds();
+    const CAMPAIGN_NAME = MODE.concat("_").concat(DATETIME);
+    const RANDOM_STRING = "_".concat(
+      Math.floor(Math.random() * 1000000 + 1).toString()
+    );
+    const SUBJECT_NAME = "sub_".concat(DATETIME).concat(RANDOM_STRING);
+    const MSG_CONTENT = Cypress.env("MSG_CONTENT").concat(RANDOM_STRING);
+
+    const OTP_SUBJECT = Cypress.env("OTP_SUBJECT");
+    const EMAIL = Cypress.env("EMAIL");
+    const MAIL_SENDER = Cypress.env("MAIL_SENDER");
+    const WAIT_TIME = Cypress.env("WAIT_TIME");
+
+    //write csv test file
+    // check if not exist first
+    cy.task("readFileMaybe", CSV_FILENAME).then((res) => {
+      if (!res) {
+        const CSV_CONTENT = "recipient,name\n" + EMAIL + ",postman";
+        cy.writeFile("cypress/fixtures/".concat(CSV_FILENAME), CSV_CONTENT);
+      }
+    });
+
+    //log in via OTP
```

Comment:
haha should have space after `//`

Reply:
got a few more later with the same issue

Reply:
fixed